src/Analyzers/CSharp/CodeFixes/UseImplicitOrExplicitType/UseImplicitTypeCodeFixProvider.cs

```diff
@@ -48,19 +50,21 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
 
             foreach (var diagnostic in diagnostics)
             {
-                var node = root.FindNode(diagnostic.Location.SourceSpan, getInnermostNodeForTie: true);
-                ReplaceTypeWithVar(editor, node);
+                var node = root.FindNode(diagnostic.Location.SourceSpan, getInnermostNodeForTie: true) as TypeSyntax;
+                if (node != null)
```

Comment:
When do we hit the null path? Is there a test for this?

Reply:
no.  you're right.  this isn't needed. t hs has to be safe given teh construction of the diagnostic